src/api/v3/feedback/event_feedback.py

```diff
@@ -5,9 +5,15 @@
 
 
 @feedback_handler
-# pylint: disable=unused-argument
 def event_feedback(data, region, language, comment, emotion, is_technical):
+    return event_feedback_internal(
+        data, region, language, comment, emotion, is_technical
+    )
+
+
+def event_feedback_internal(data, region, language, comment, emotion, is_technical):
     event_slug = data.get("slug")
+    print(event_slug)
```

Comment:
Do we need the print?

Reply:
If yes, please consider using `logger.debug()`.